libs/jsonschema/schema.go

```diff
@@ -6,14 +6,17 @@ import (
 	"os"
 	"regexp"
 	"slices"
-	"strings"
 
 	"github.com/databricks/cli/internal/build"
 	"golang.org/x/mod/semver"
 )
 
 // defines schema for a json object
 type Schema struct {
+	// Definitions that can be reused and referenced throughout the schema. The
+	// syntax for a reference is $ref: #/$defs/<path.to.definition>
+	Definitions any `json:"$defs,omitempty"`
```

Comment:
This should always be a `map[string]any` right?

Reply:
Yeah, good point. Done.